src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/SpecimenReferenceByServiceProviderTest.java

```diff
@@ -0,0 +1,42 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import org.apache.commons.lang3.StringUtils;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+
+public class SpecimenReferenceByServiceProviderTest {
+
+    @Test
+    void toEdifactTest() {
+        var edifact = new SpecimenReferenceByServiceProvider("CH000064LX").toEdifact();
+
+        Assertions.assertThat(edifact).isEqualTo("RFF+STI:CH000064LX'");
```

Comment:
Static import the method (and siblings) so you don't need to qualify it each time

Reply:
Same as above: 6a43c8a60f62d662721e875fdfed47e114992f77
Thanks!